code/datums/ai/basic_mobs/basic_subtrees/call_reinforcements.dm

```diff
@@ -0,0 +1,48 @@
+#define REINFORCEMENTS_COOLDOWN (30 SECONDS)
+
+/// Calls all nearby mobs that share a faction to give backup in combat
+/datum/ai_planning_subtree/call_reinforcements
+	/// Text to say when calling reinforcements
+	var/call_say
+	/// Text to emote when calling reinforcements
+	var/call_emote = "cries for help!"
```

Comment:
These should probably also just be in the blackboard instead of vars (although they can have default fallbacks)